CONTRIBUTING.md

```diff
@@ -205,36 +208,37 @@ To make a release as a core contributor, you will need:
 Before release:
 - If necessary, make and merge a PR with any last minute housekeeping items.
 - Turf's documentation (README.md files) should already be up to date as it is generated automatically on commit from JSDoc comments in source files.
-- Review PR's and decide the new version number to be published (for example 7.0.1 or 7.1.0).
-  - If there are breaking changes, then it should be a major version bump, e.g. 7.x.x to 8.0.0.  This project follows [semantic versioning](https://semver.org/).
+- Review PR's and decide the new version number to be published. See [semantic versioning](https://semver.org/).
 
-Run the release commands, replace `7.0.0` with your version number
-- fetch the latest code from remote origin
-  - `git fetch origin`
+Run the following release commands, replacing `7.0.0` with your version number:
 
-- create a release branch, replace mf with your initials to make it clear whose branch it is.
+- create a release branch, replacing `mf` with your initials to make it clear who created the branch.
```

Comment:
do you want to change `mf` to `xx` in the docs or something?

Reply:
yeah I can do that, make it stand out